### DISCUSSION TITLE
TheHive WEBHOOK Worflow - SYNTAXERROR: UNEXPECTED TOKEN < IN JSON AT POSITION 0

### DISCUSSION THREAD

**Discussion by Shironoh** [Shuffle/Shuffle#887] [Q&A]
Hello,

I'm getting SYNTAXERROR: UNEXPECTED TOKEN < IN JSON AT POSITION 0 when i'm working with a basic Worflow Webhook ==> Repeater ($exec value).

Regarding TheHive events format it seems that it's in a good json format. 

![image](https://user-images.githubusercontent.com/108070991/175298257-73b1c9fe-01df-4164-bca4-b123c2bb6cae.png)

Kind regards,

**Comment by Shironoh** [comment]


**Reply by Shironoh** [reply]
![image](https://user-images.githubusercontent.com/108070991/175307312-4e59c79d-f700-41f9-9f23-74a669493f7b.png)

**Reply by Shironoh** [reply]
"Results for Execution Argument":{11 items
"operation":"create"
"details":{10 items
"_id":"~655392872"
"_type":"Task"
"_createdBy":"test"
"_createdAt":1655968413724
"title":"Retune alert detection"
"group":"default"
"status":"Waiting"
"flag":false
"order":0
"extraData":{}0 items
}
"objectType":"case_task"
"objectId":"~655392872"
"base":false
"startDate":1655968413735
"rootId":"~245788824"
"requestId":"2034422ff97a42fc:-141004bd:1818bd1bfe2:-8000:4709"
"object":{11 items
"id":"~655392872"
"_id":"~655392872"
"createdBy":"test"
"createdAt":1655968413724
"_type":"case_task"
"title":"Retune alert detection"
"group":"default"
"status":"Waiting"
"flag":false
"order":0
"case":{25 items
"_id":"~245788824"
"id":"~245788824"
"createdBy":"test"
"updatedBy":NULL
"createdAt":1655968413717
"updatedAt":NULL
"_type":"case"
"caseId":12
"title":"False positive case test"
"description":"False positive template"
"severity":2
"startDate":1655968380000
"endDate":NULL
"impactStatus":NULL
"resolutionStatus":NULL
"tags":[]0 items
"flag":false
"tlp":2
"pap":2
"status":"Open"
"summary":NULL
"owner":"test"
"customFields":{}0 items
"stats":{}0 items
"permissions":[]0 items
}
}
"organisationId":"~20480"
"organisation":"OPS"
}

**Comment by frikky** [comment]
Made this into an issue at https://github.com/Shuffle/Shuffle/issues/889
